src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -659,9 +692,12 @@ export class CalciteCombobox {
   //--------------------------------------------------------------------------
 
   renderChips(): VNode[] {
-    const { activeChipIndex, scale } = this;
-    return this.selectedItems.map((item, i) => {
+    const { activeChipIndex, scale, selectionMode } = this;
+    return (this.selectedItems || []).map((item, i) => {
```

Comment:
nitpick: maybe selectedItems should be set as empty array so that the `or` isn't needed here.